packages/alert-message-react/documentation/AlertMessageExample.tsx

```diff
@@ -19,7 +19,7 @@ const getTypeOfBox = (typeofBox?: string) => {
 };
 
 export const AlertMessageExample: React.FC<ExampleComponentProps> = ({ boolValues, choiceValues }) => {
-    const C = getTypeOfBox(choiceValues ? choiceValues["Type"] : "");
+    const C = getTypeOfBox(choiceValues ? choiceValues["Variant"] : "");
```

Comment:
Konsoliderer på ett navn for varianter. Oppdaget at vi bruker Type om meldingsbokser og Variant om alt annet.